dll/shellext/netplwiz/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+
+spec2def(netplwiz.dll netplwiz.spec ADD_IMPORTLIB)
```

Comment:
You don't need the `ADD_IMPORTLIB` if you are calling it using `LoadLibrary / GetProcAddress`